Yoast/wordpress-seo
# add folder and researcher for Greek

## Context 


*

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds a language folder and Researcher for Greek. 

## Relevant technical choices:

*

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Build the free plugin
* Enable the GREEK_SUPPORT feature by adding define( 'YOAST_SEO_GREEK_SUPPORT', true ); in `basic-wordpress-config.php` file
* Set your site to Greek `Ελληνικά`
* Create/open a post
* In the console, enter `window.yoast.Researcher` and confirm that it returns as shown below (the Researcher file should start with `el`):
<img width="534" alt="Screenshot 2021-09-23 at 14 08 49" src="https://user-images.githubusercontent.com/90325221/134504990-6cecc2c0-f919-49f0-861c-ff1ce39ba561.png">

* Disable the GREEK_SUPPORT by adding this define( 'YOAST_SEO_GREEK_SUPPORT', false ); in `basic-wordpress-config.php` file
* Reload your page
* Confirm that you get the default Researcher by entering `window.yoast.Researcher` in the console:
<img width="535" alt="Screenshot 2021-09-23 at 14 10 39" src="https://user-images.githubusercontent.com/90325221/134505311-cfba9380-48c4-4b48-bc29-27e31a9772e9.png">


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* * Install and activate the free plugin
* Enable the GREEK_SUPPORT feature by adding define( 'YOAST_SEO_GREEK_SUPPORT', true ); in `basic-wordpress-config.php` file
* Set your site to Greek `Ελληνικά`
* Create/open a post
* In the console, enter `window.yoast.Researcher` and confirm that it returns as shown below (the Researcher file should start with `el`):
<img width="534" alt="Screenshot 2021-09-23 at 14 08 49" src="https://user-images.githubusercontent.com/90325221/134504990-6cecc2c0-f919-49f0-861c-ff1ce39ba561.png">

* Disable the GREEK_SUPPORT by adding this define( 'YOAST_SEO_GREEK_SUPPORT', false ); in `basic-wordpress-config.php` file
* Reload your page
* Confirm that you get the default Researcher by entering `window.yoast.Researcher` in the console:
<img width="535" alt="Screenshot 2021-09-23 at 14 10 39" src="https://user-images.githubusercontent.com/90325221/134505311-cfba9380-48c4-4b48-bc29-27e31a9772e9.png">


## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Documentation


## Quality assurance


Fixes #
https://yoast.atlassian.net/browse/LINGO-1044
